<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VfkMainWindow</class>
 <widget class="QMainWindow" name="vfkMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prohlížeč VFK</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="searchPage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="searchCombo"/>
         </item>
         <item>
          <widget class="QStackedWidget" name="searchForms">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QScrollArea" name="scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>98</width>
                  <height>28</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="VlastniciSearchForm" name="vlastniciSearchForm" native="true"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QScrollArea" name="scrollArea_3">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>98</width>
                  <height>28</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="ParcelySearchForm" name="parcelySearchForm" native="true"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QScrollArea" name="scrollArea_2">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_3">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>98</width>
                  <height>28</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="BudovySearchForm" name="budovySearchForm" native="true"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_4">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QScrollArea" name="scrollArea_4">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_4">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>156</width>
                  <height>235</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="JednotkySearchForm" name="jednotkySearchForm" native="true"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>Hledej</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="text">
             <string>&lt;-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forthButton">
            <property name="text">
             <string>-&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="latexExportButton">
            <property name="text">
             <string>LaTeX</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="VfkTextBrowser" name="vfkBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport"/>
   <addaction name="actionVyhledavani"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionVyhledavani">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vyhledávání</string>
   </property>
   <property name="toolTip">
    <string>Vyhledat</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VfkTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>vfktextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>ParcelySearchForm</class>
   <extends>QWidget</extends>
   <header>parcelysearchform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlastniciSearchForm</class>
   <extends>QWidget</extends>
   <header>vlastnicisearchform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BudovySearchForm</class>
   <extends>QWidget</extends>
   <header>budovysearchform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JednotkySearchForm</class>
   <extends>QWidget</extends>
   <header>jednotkysearchform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
